pkg/tools/grpcutils/health.go

```diff
@@ -33,3 +36,22 @@ func RegisterHealthServices(s *grpc.Server, services ...interface{}) {
 		}
 	}
 }
+
+// WaitStatusServing wait status serving for each passed service
+func WaitStatusServing(ctx context.Context, client grpc_health_v1.HealthClient, services ...interface{}) error {
```

Comment:
Why not just use [grpc.WaitForReady](https://godoc.org/google.golang.org/grpc#WaitForReady) ?

Reply:
his util method can be used for k8s probes, but actually, in this PR we can simply use grpc.WaitForReady. 
Fixed.